docs/advanced/retries.rst

```diff
@@ -0,0 +1,57 @@
+.. _retries:
+
+=======
+Retries
+=======
+
+The webpages of some websites can be unreliable. For example, sometimes
+a request can get a response that may only include a part of the data to be
+extracted, or no data at all, but sending a follow-up, identical request can
+get you the expected data.
+
+You can write your page object so that it raises
```

Comment:
minor: should we use "Page Object" (capitalized first letters) to be consistent with how it's being written in other parts of the doc like https://web-poet.readthedocs.io/en/stable/advanced/additional-requests.html ? or the other way around?

Reply:
My thinking is that Page Object is correct when referring to the design pattern, as it is a proper noun, but for the actual objects “page object” would be the right capitalization. So I don’t see “Page Object” as accurate in this context.

The documentation is already quite inconsistent in this regard (see https://web-poet.readthedocs.io/en/stable/intro/from-ground-up.html#page-objects), so I felt entitled to pick the lowercase option.

I am in favor of making the wording consistent across the documentation, even if that means capitalizing the term always, but I think it would be best done in a separate pull request.